components/Navbar.tsx

```diff
@@ -85,6 +85,7 @@ export default function WithSubnavigation({ onRegisterOpen }: { onRegisterOpen:
               as="a"
               onClick={onRegisterOpen}
               size="lg"
+              cursor="pointer"
```

Comment:
I added this CSS property here bc it is inconsistent/not typical UX to not have cursor: pointer for links/buttons